lib/string/strcpy/strtcpy.h

```diff
@@ -34,8 +34,10 @@ strtcpy(char *restrict dst, const char *restrict src, size_t dsize)
 	bool    trunc;
 	size_t  dlen, slen;
 
-	if (dsize == 0)
+	if (dsize == 0) {
+		errno = EOVERFLOW;
```

Comment:
Your commit says "Report EOVERFLOW for symmetry with stprintf()", but the next commit
changes vstprintf to return E2BIG if it gets EOVERFLOW from vsnprintf.  Do you mean a
different stprintf, of did you intend to change patch 13/13 ?

Reply:
I meant for symmetry with the EOVERFLOW that we return if `(size <= 0 || size > INT_MAX)`.  I'll reorder the commits to make it clear.